### ISSUE TITLE
[Bug Report] CustomErrorPageAnalyzer fails with an exception

### ISSUE THREAD

**Issue by mixdf** [enlightn/enlightn#147]
### Versions

* PHP version: 8.2
* Laravel version: 9.52.4
* Enlightn version: 2.3.0
* Enlightn Pro version (if applicable):

### Description
Running the analyzer fails with the following message:
```
Check 22/64: Your application defines custom error page views. Exception
Undefined variable $exception (View: /var/www/resources/views/pages/errors/404.blade.php)
```

### Steps to Reproduce
1. Change the default errors view namespace using `View::replaceNamespace('errors', resource_path('views/custom/errors')`
2. Edit one of the custom error pages and use `$exception` variable (it is available in the view)
3. Run the analyzer
4. See it fails with an exception

**Expected behavior:**
It should recognize the custom error page and doesn't throw an exception.

**Actual behavior:**
A new check was added to the analyzer, but the implementation is wrong. The exception instance and the error bag should be passed into the error page. It also doesn't work if the default namespace is modified (for instance if the error pages are on `resources/views/custom/errors` instead of `resources/views/errors` directory).

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by paras-malhotra** [comment]
@mixdf we've fixed the custom error namespace in #148. Can you confirm?

For the `$exception` variable, I'm not sure why it isn't passed. `abort(404)` does the exact same thing - it throws a `NotFoundHttpException`. Can you share some code to reproduce the error on a standard Laravel app?

https://github.com/laravel/framework/blob/8427a4cd9399b711c73ca00cd3857d7470908cdf/src/Illuminate/Foundation/Application.php#L1218-L1225

**Comment by mixdf** [comment]
Hey @paras-malhotra, thanks for the response. Let me try to check and get back to you during the weekend. I can open a PR as well to fix it if you think that'll help.
